src/form/FormInput.js

```diff
@@ -122,12 +122,13 @@ styles = StyleSheet.create({
     ...Platform.select({
       android: {
         height: 46,
+        width: '100%',
```

Comment:
@Draer Instead of this change this to `width: width - 25` you can also move `width: width` before the ...Platform.select so you don't have to define in ios